src/components/card/CardImage.tsx

```diff
@@ -52,12 +48,14 @@ class CardImage extends PureComponent<Props> {
   constructor(props: Props) {
     super(props);
 
+    this.styles = createStyles(props);
+
+    if (props.imageSource) {
+      LogService.deprecationWarn({component: 'CardImage', oldProp: 'imageSource', newProp: 'source'});
+    }
     if (props.borderRadius) {
-      LogService.warn(
-        'uilib: Please stop passing borderRadius to Card.Image, it will get the borderRadius from the Card'
-      );
+      LogService.deprecationWarn({component: 'CardImage', oldProp: 'borderRadius'});
```

Comment:
That's a good idea to add this option, but the current message expects the `newProp` (actually this is my bad, I need to fix LogService typing).
You can use the `warn` method for now.

Reply:
My suggestion is that we change the LogService deprecationWarn's massage to accommodate this scenario. What do you think?

Reply:
Added this option in the other PR